WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -329,7 +329,14 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
         }
 
         let onDismissQuickStartPrompt: (Blog, Bool) -> Void = { [weak self] blog, _ in
-            self?.onDismissQuickStartPrompt(for: blog, onDismiss: onDismiss)
+            guard let self = self else {
+                // If self is nil the user will be stuck on the login and not able to progress
+                // Trigger a fatal so we can track this better in Sentry.
+                // This case should be very rare.
+                fatalError("Could not get a reference to self when selecting the blog on the login epilogue")
```

Comment:
I think the best option, for now, is the `fatalError`. If we see too many occurrences of it we can probably try that:

```
WordPressAppDelegate.shared?.windowManager.dismissFullscreenSignIn(blogToShow: blog)
```

This will force the selected blog to be displayed anyway.

Reply:
My concern with this approach is the unexpected state the app would be in and the other potential crashes or problems that could occur because of it.